cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -113,6 +113,9 @@ SECTIONS
         KEEP (*(.isr_stack))
         . = ALIGN(8);
         _estack = .;
+        KEEP (*(.puf_stack))
```

Comment:
Basically yes. I didn't consider the heap section but seems like a good candidate. I'll figure it out.

Reply:
If you place the puf section first there is less risk that it will be relocated when flashing new firmwares of an application. This reduces the risk of false negatives for the soft reset detection during OTA upgrades.